src/test/java/de/retest/recheck/ignore/JSFilterImplTest.java

```diff
@@ -16,9 +17,11 @@
 
 class JSFilterImplTest {
 
+	Path ctorArg = new File( "unused" ).toPath();
```

Comment:
You should be able to reference a temporary directory. Otherwise why not just `Paths.get("unused")`?
```suggestion
        @TempDir
	Path ctorArg;
```

Reply:
I'd rather not introduce an unused tmp dir. So `Paths.get()` then.